src/membraneWebRTC.ts

```diff
@@ -144,6 +144,11 @@ export interface Callbacks {
    */
   onJoinError?: (metadata: any) => void;
 
+  /**
+   * Called every time a local peer is removed by the server.
+   */
+  onRemoved?: (reason: string) => void;
```

Comment:
I assume it was just moved here from somewhere, because it makes more sense here?

Reply:
Yeap, it's close to `onJoinSuccess` and `onJoinError`